bin/node/runtime/pangolin/Cargo.toml

```diff
@@ -13,6 +13,7 @@ version    = "2.4.0"
 # crates
 array-bytes       = { version = "1.1.0" }
 codec             = { package = "parity-scale-codec", version = "2.1.1", default-features = false }
+evm-crate         = { default-features = false, version = "0.25.0", features = ["with-codec"], package = "evm" }
```

Comment:
Any reason rename evm to evm-crate?

Reply:
O, that's because the `crate evm` will conflict with the`runtime mod evm`.